e2e/__projects__/style/test.js

```diff
@@ -22,43 +22,59 @@ function mount(Component, props, slots) {
   createApp(Parent).mount(el)
 }
 
-xtest('processes Less', () => {
+test('processes Less', () => {
   mount(Less)
-  // expect(wrapper.is('div')).toBeTruthy()
-  // expect(wrapper.vm.$style.a).toEqual('a')
+  expect(document.getElementById('app').innerHTML).toEqual(
+    '<div><div class="a">a</div><div class="b">b</div></div>'
+  )
 })
 
-xtest('processes PostCSS', () => {
+test('processes PostCSS', () => {
   mount(PostCss)
+  expect(document.getElementById('app').innerHTML).toEqual(
+    '<section><div class="c"></div><div class="d"></div></section>'
+  )
   // expect(wrapper.is('section')).toBeTruthy()
   // expect(wrapper.vm.$style.a).toEqual('a')
   // expect(wrapper.vm.$style.b).toEqual('b')
 })
 
-xtest('processes Sass', () => {
+test('processes Sass', () => {
   mount(Sass)
+  expect(document.getElementById('app').innerHTML).toEqual(
+    '<div><div class="a"></div><div class="b"></div><div class="c"></div><div class=""></div><div class="e"></div></div>'
```

Comment:
Some css property is not moduled (For example d), so global class is not rendered. 

```html
<style module lang="sass">
@import "~__styles/sass-a"

.c
  background-color: red
</style>

<style lang="sass">
.d
  background-color: blue
</style>

<style lang="sass" module themed>
.e
  background-color: blue
</style>
```